### PR TITLE
[fix bug 1363310] Add localized redesigned site footer

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Moves all pages and locales to a unified, redesigned footer.

## Bugzilla link
- https://bugzilla.mozilla.org/show_bug.cgi?id=1363310

## Testing
Demo: https://www-demo4.allizom.org/en-US/

- Test Pebbles (both adaptive and fluid pages, RTL)
- Test Sandstone (responsive, non-responsive, old IE styles, RTL)

## Checklist
- [x] Requires l10n changes.
- [x] Related functional & integration tests passing.

### CODE REVIEW

**Comment by alexgibson** [comment]
I'm confirming that strings are final in the bug, will update here once I get word.

**Comment by alexgibson** [comment]
@peiying2 @flodolo - OK, strings here are now final. The footer uses a shared .lang file, so I understand we'll have to wait for these to be translated.

This branch is up on demo: https://www-demo4.allizom.org/en-US/

Please let me know if you need anything or have any questions.

**Comment by flodolo** [comment]
@peiying2 
For reference: go through [this file](https://github.com/alexgibson/bedrock/blob/58ef9a45737dc9a8ee77c8eae4cd21351d15104e/bedrock/base/templates/includes/site-footer.html), and see how many strings we're missing in main.lang.

**Comment by flodolo** [comment]
(sorry, misclicked in the UI…)

**Comment by peiying2** [comment]
@alexgibson localization of the strings started last Friday.  Email and reminders were sent on the deadline that the unified footer will go on production on June 16 . Here is the current status: https://l10n.mozilla-community.org/langchecker/?locale=all&website=0&file=main.lang.  The major languages are in good shape.  At this point, l10n shouldn't be your blocker with the push.  Thanks!

**Comment by alexgibson** [comment]
> @alexgibson localization of the strings started last Friday. Email and reminders were sent on the deadline that the unified footer will go on production on June 16 . Here is the current status: https://l10n.mozilla-community.org/langchecker/?locale=all&website=0&file=main.lang. The major languages are in good shape. At this point, l10n shouldn't be your blocker with the push. Thanks!

Great, thanks @peiying2!

**Comment by alexgibson** [comment]
@jpetto @craigcook this one touches a lot of pages, so whomever has time a review would be appreciated.

**Comment by alexgibson** [comment]
@craigcook @jpetto - r?

File: media/css/pebbles/components/_footer.scss [hunk 1]
```diff
@@ -5,33 +5,61 @@
 @import '../includes/lib';
 
 #colophon {
+    @include open-sans;
     @include font-size(14px);
-    background: #fff;
-    color: $color-text-primary;
+    background: #000;
+    color: #fff;
     line-height: 1.5;
```
Comment (jpetto) [inline media/css/pebbles/components/_footer.scss]: Is this line height meant to remain at 1.5? It is consistent with the rest of the site, but it makes the footer pretty tall... Not a blocker - just checking.

File: media/css/sandstone/sandstone.less [hunk 2]
```diff
@@ -740,253 +740,195 @@ nav.menu-bar {
...
 
 #colophon {
-    .font-size(@baseFontSize);
-    background: #fff;
-    color: @textColorTertiary;
-    line-height: 1.3;
+    .font-size(14px);
+    background: #000;
+    color: #fff;
+    line-height: 1.5;
```
Comment (jpetto) [inline media/css/sandstone/sandstone.less]: Same question on line height here.

File: bedrock/firefox/templates/firefox/product-base.html [hunk 3]
```diff
@@ -8,16 +8,6 @@
...
-{% block site_footer %}
-  {% with footer_class='dark' %}
-    {% if LANG.startswith('en-') %}
-      {% include 'includes/site-footer-new.html' %}
-    {% else %}
-      {% include 'includes/site-footer.html' %}
-    {% endif %}
-  {% endwith %}
-{% endblock %}
-
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/product-base.html]: Hmm, you're quite right. I wonder if this got orphaned during a rebase? it certainly seems unused, so I'll delete it 👍

File: media/css/pebbles/components/_footer.scss (hunk 1 above)
Comment (alexgibson) [inline media/css/pebbles/components/_footer.scss]: Hmm, this was the spacing in the mockup but I do agree it seems tall. I think we could maybe drop it to 1.3 without folks really noticing. Thoughts?

Comment (jpetto) [inline media/css/pebbles/components/_footer.scss]: 1.3 is what I tried in dev tools and it looked good to me. 👍

**Comment by alexgibson** [comment]
@jpetto thanks for the review, updated and squashed!

**Pull request merged**